README.md

```diff
@@ -266,7 +331,7 @@ As already discussed, employees can get restricted stock, stock options, or RSUs
 - 🔹 If you are granted ISOs or NSOs at a low strike price, and the bargain element is zero, then you may be able to exercise at a reasonable price without triggering taxes at all. So assuming the company allows it, it makes sense to early exercise immediately (buying most or all of the shares, even though they’re not vested yet) and simultaneously file an 83(b) election.
 - 🔸☝️ Section 83(b) elections are elections to be taxed on the receipt of property even though you might have to forfeit or give back the property to the company. You can make an election on the receipt of stock, but you cannot make the election on the receipt of an option or an RSU because options and RSUs are not considered property for purposes of Section 83(b).
-- 🔸🌪 ISOs are often preferred by startups, as they're supposedly better for employees from a tax perspective. This assumes that (1) AMT won’t be triggered and (2) you’ll get low long-term capital gains rate by holding the stock for the appropriate holding periods. However, often you either run afoul of the AMT trap, or don’t hold the stock long enough with the complicated 1 year + 2 year requirement, or the spread at exercise is small or zero, so the difference wouldn’t matter anyway. NSOs do have a slightly higher tax because of the employment taxes. Overall, it’s not clear the ISO is that much better for employees, so [many](http://www.startuplawblog.com/2010/08/11/top-reasons-nqos-over-isos/) [people](http://www.business2community.com/finance/nsos-better-isos-0826167#fz1HTCiOQxRyTr62.97) argue for NSOs instead.
+- 🔸🌪 ISOs are often preferred by startups, as they’re supposedly better for employees from a tax perspective. This assumes that (1) AMT won’t be triggered and (2) you’ll get low long-term capital gains rate by holding the stock for the appropriate holding periods. However, often you either run afoul of the AMT trap, or don’t hold the stock long enough with the complicated 1 year + 2 year requirement, or the spread at exercise is small or zero, so the difference wouldn’t matter anyway. NSOs do have a slightly higher tax because of the employment taxes. Overall, it’s not clear the ISO is that much better for employees, so [many](http://www.startuplawblog.com/2010/08/11/top-reasons-nqos-over-isos/) [people](http://www.business2community.com/finance/nsos-better-isos-0826167#fz1HTCiOQxRyTr62.97) argue for NSOs instead.
```

Comment:
I know this is covered at another point, but what are the appropriate holding periods for long-term cap gains?